extensions/checksum/examples/sentinel1.json

```diff
@@ -44,9 +44,7 @@
       "href": "./measurement/s1a-ew-grd-hh-20181103t235855-20181103t235955-024430-02ad5d-001.tiff",
       "title": "Measurements",
       "type": "image/tiff",
-      "sar:bands": [
-        0
-      ],
+      "sar:polarizations": ["HH"],
```

Comment:
Please open an issue on asset-specific metadata before we merge this so that we don't forget to discuss it for 0.9.